### PR TITLE
add reservation confirmation email

### PR DESCRIPTION
**Pull request by Joannitka-pl**

### CODE REVIEW

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'joannitka.pl2@gmail.com'
```
Comment (wmaciejak) [inline app/mailers/application_mailer.rb:2]: Do not put your private address of emails into repository 🙏

File: config/environments/development.rb [hunk 2]
```diff
@@ -50,6 +52,16 @@
   # Highlight code that triggered database queries in logs.
   config.active_record.verbose_query_logs = true
 
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+    port: 587,
+    address: 'smtp.gmail.com',
+    :user_name            => 'joannitka.pl2@gmail.com',
```
Comment (wmaciejak) [inline config/environments/development.rb:60]: Move it to new env variable like `ENV["MAILER_USER_NAME"]` or something like this

File: app/mailers/client_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class ClientMailer < ApplicationMailer
```
Comment (wmaciejak) [inline app/mailers/client_mailer.rb:1]: Does this mailer generally work for you?

Comment (Joannitka-pl) [reply app/mailers/client_mailer.rb:1]: Yes, I've received generated emails.

File: config/environments/development.rb (hunk 2 above)
Comment (Joannitka-pl) [reply config/environments/development.rb:60]: Sure, will do that. Thank you :)

File: app/mailers/client_mailer.rb (hunk 3 above)
Comment (wmaciejak) [reply app/mailers/client_mailer.rb:1]: Awesome 👌

File: app/mailers/application_mailer.rb (hunk 1 above)
Comment (Joannitka-pl) [reply app/mailers/application_mailer.rb:2]: Of course 🙈😅

**Pull request merged**